apps/pie-docs/README.md

```diff
@@ -31,6 +31,22 @@
 - Our unit tests are written using [Jest](https://jestjs.io/)
 - Our accessibility tests are written using [Axe with Playwright](https://playwright.dev/docs/accessibility-testing)
 
+## Drafts
+
+When building a page that is not yet ready for production we can mark the page as a `draft` by adding `draft: true` to the page front matter. This will allow 11ty to build the page during development mode but will exclude the page from builds during production.
+
+```
+---
+title: Test Page
+description: Test page
+draft: true
+---
+```
+
+To create a new section that will be in draft mode, each page in the section will need to have `draft: true` added to the page front matter.
+
+> When running `yarn test:generate-routes`, the routes are generated based off of what is in the `/dist/` folder. If you need to regenerate routes while we have a section in drafts you will first need to run `yarn build` and then run `yarn test:generate-routes` to generate the routes that will be used in production.
```

Comment:
Instead of documenting this, have you considered simply adding `yarn build` to the `test:generate-routes` command? This piece of documentation is prone to getting outdated.

Reply:
We've updated `turbo.json` to now run `yarn build` before generating routes 🥳 
https://github.com/justeattakeaway/pie/pull/473/files#diff-f8de965273949793edc0fbfe249bb458c0becde39b2e141db087bcbf5d4ad5e3R71

Reply:
Nice. Does that work also when you just run `yarn test:generate-routes`?